src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -33,7 +36,22 @@ export class RadioButton extends Widget {
 		this.label = opts.label;
 		this.enabled = opts.enabled || true;
 		this.checked = opts.checked || false;
-		this.onclick(this.inputElement, () => this._onClicked.fire());
+		this.onclick(this.inputElement, () => {
+			this._onClicked.fire();
+			if (this.name) {
+				const buttonGroup = document.getElementsByName(this.name);
+				buttonGroup.forEach((button) => {
+					const event = document.createEvent('HTMLEvents');
+					event.initEvent('change', true, true);
+					button.dispatchEvent(event);
+				});
+			}
+		});
+		this.inputElement.addEventListener('change', () => {
+			if (this.inputElement.checked !== this._internalCheckedStateTracker) {
+				this.checked = this.inputElement.checked;
```

Comment:
I would suggest putting this check in the checked setter instead so that we only fire the event when the value actually changes. Then this event handler would just be simply 
`this.inputElement.addEventListener('change', () => this.checked = this.inputElement.checked);`